app/views/pages/home/_confidence.html.erb

```diff
@@ -1,51 +1,51 @@
 <div class="govuk-width-container">
-  <div class="ncce-confidence__header govuk-heading-m">
+  <div class="ncce-confidence__header govuk-heading-s">
     <h1><%= t("pages.home.confidence.heading") %></h1>
   </div>
 
   <div class="govuk-main-wrapper ncce-confidence">
     <div class="ncce-confidence__card">
-      <%= image_pack_tag 'media/images/home/card_courses.jpg', loading: 'lazy', class: 'ncce-confidence__icon', alt: '' %>
+      <%= link_to image_pack_tag('media/images/home/card_courses.jpg', loading: 'lazy', class: 'ncce-confidence__icon', alt: ''), courses_path %>
```

Comment:
For these links, if you move the class onto the link, rather than the image. The original justify-self method will work for the alignment on mobile view

Reply:
Moved the class onto all the links